### DISCUSSION TITLE
Streaming w azure media services.

### DISCUSSION THREAD

**Discussion by galante13** [AzureCommunityPL/group#398] [Pomoc] [answered]
Zacząłem niedawno kombinować z azure media services i próbuję dodać overlaye do live eventu. Po krótce mam jako input stream rtmp i chcę w locie dodawać np. overlay w postaci znaku wodnego albo jakieś inne zdjęcie. W necie znajduję tylko sample dodawania takich rzeczy do video, czy to oznacza, że aby to zrobić w live evencie, trzeba działać na output assecie, czy można to jakoś zrobić w locie przez np. Live event type None? Macie jakieś wskazówki, gdzie znaleźć odpowiedź, mogłem się już pogubić w dokumentacji i może nie widzę odpowiedzi...?

**Comment by smereczynski** [comment]
Nie patrzyłem na temat od dawna - dobre dwa lata - ale jak przewijało mi się to wtedy, to była mowa o OBS Studio na maszynie wirtualnej, gdzie dodawany jest overlay i dalej stream do AMS. Być może nie jest to odpowiedź na Twoje pytanie - bo nie wiem czy overlay jest dziś w AMS możliwy - ale jest to głos mówiący jak można to zrealizować.

**Comment by galante13** [comment, marked as answer]
Dzięki za odpowiedź!

Temat przebadany, również z supportem msu i rzeczywiście nie da się dodawać on-fly overlayi.
